### PR TITLE
ADR-1590: request payments within a range

### PR DESCRIPTION
**Pull request by sini-george**

### CODE REVIEW

File: app/uk/gov/hmrc/alcoholdutyaccount/connectors/FinancialDataConnector.scala [hunk 1]
```diff
@@ -95,8 +95,11 @@ class FinancialDataConnector @Inject() (config: AppConfig, implicit val httpClie
       "customerPaymentInformation" -> false.toString
     )
 
-  private def getOnlyOpenItemsFalseParameters(year: Int): Seq[(String, String)]               =
-    getBaseQueryParams(false) ++ Seq("dateFrom" -> s"$year-01-01", "dateTo" -> s"$year-12-31")
+  private def getOnlyOpenItemsFalseParameters: Seq[(String, String)] =
+    getBaseQueryParams(false) ++ Seq(
+      "dateFrom" -> "2024-10-31",
```
Comment (ajchilts) [inline app/uk/gov/hmrc/alcoholdutyaccount/connectors/FinancialDataConnector.scala:100]: Should we add these to app config so we have some degree of control over them?

**Comment by ajchilts** [comment]
PR builder has failed.

File: app/uk/gov/hmrc/alcoholdutyaccount/connectors/FinancialDataConnector.scala (hunk 1 above)
Comment (sini-george) [reply app/uk/gov/hmrc/alcoholdutyaccount/connectors/FinancialDataConnector.scala:100]: Sure, I can add them, I didn't add them earlier as I thought these were temporary changes and the pagination story would change this anyway..

**Review by ajchilts — CHANGES_REQUESTED** [review]
Happy to approve once comments addressed and build green.

**Review by davidfes — APPROVED** [review]

**Pull request merged**